scalafix-tests/unit/src/test/scala/scalafix/tests/rule/MavenFuzzSuite.scala

```diff
@@ -158,7 +160,7 @@ class MavenFuzzSuite extends AnyFunSuite with DiffAssertions {
         .withPaths(paths.asJava)
         .withRules(List(rule).asJava)
         .withClasspath(classfiles.asJava)
-      // .withMode(ScalafixMainMode.CHECK)
+        .withMode(ScalafixMainMode.CHECK)
```

Comment:
What does this mean?

Reply:
there are 3 modes. this one only checks and doesn't write to file.